var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -188,12 +188,19 @@ def install(self, spec, prefix):
                 "HWLOC_INCLUDE_DIR = {0}".format(spec["hwloc"].prefix.include),
                 "config.mk",
             )
-            filter_file(
-                "^#HWLOC_LIB_DIR.*",
-                "HWLOC_LIB_DIR = {0}".format(spec["hwloc"].prefix.lib),
```

Comment:
You should use `spec["hwloc"].libs` to obtain what you call `ll`. That, hopefully, would also simplify the changes here.

Reply:
I fixed it in https://github.com/spack/spack/pull/39659/commits/d18251fa19bb7ef6e6d4d8b88748a6e0aee042a9

And this is working and future-proof since spack already had the "wrong" value in `spec["hwloc"].prefix.lib`?

Reply:
`prefix` is a sort of path like object pointing to the place of installation. For some reason, in the past it was decided that `.` on that object had to have path concatenation semantic, so:
```
prefix.lib
```
is "just" constructing a path starting from prefix and adding "lib".